src/less/progress-bar/base/progress-bar.less

```diff
@@ -0,0 +1,31 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.progress {
+    .customColorProperty(progress-bar-color);
+
+    border: none;
+    height: 4px;
+    margin: 8px 0;
+    @media (min-width: 601px) {
```

Comment:
By convention we group all viewport related rules in a block at the end of the file. Just makes them a little easier to find & update all in one place.